### PR TITLE
[MDM] Create a simple MDM/AppConfig/specfile.xml

### PR DESCRIPTION
**Pull request by mneuwert**
## Description
App-Config contains only following options:
- single-account: false/true
- fixed (default) server URL

## Related Issue
#236 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by michaelstingl** [comment]
@mneuwert could you add some quick notes how to use this with MDM tools? (MobileIron for example)

**Comment by mneuwert** [comment]
@michaelstingl please check if the README is fine for you?

File: MDM/AppConfig/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## AppConfig how to
+
+### Managed App Config
+Starting with iOS7, Apple has added support for managed application configuration. MDM server can push configuration to the app. The app can the access the configuration using NSUserDefaults class. The configuration is basically a key-value dictionary provided as .plist file.
+
+### AppConfig XML Schema
+The XML format developed by AppConfig community, makes it easy for developers to define and deploy app configuration. It is not only defining the configuration variables with default value but also provides a configuration UI description which can be interpreted by the tool which generates a plist file. Moreover, specfile XML is consistently supported by major EMM vendors.
+
+### Example: Deployment with Mobile Iron
```
Comment (michaelstingl) [inline MDM/AppConfig/README.md]:
```suggestion
### Example: Deployment with MobileIron
```

File: MDM/AppConfig/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+1. Open AppConfig Generator: https://appconfig.jamfresearch.com
+2. Upload a specfile.xml
+3. Change configuration options
+4. Download generated plist file (ManagedAppConfig)
+5. Open Mobile Iron Core
```
Comment (michaelstingl) [inline MDM/AppConfig/README.md]:
```suggestion
5. Open MobileIron Core
```

Comment (mneuwert) [reply MDM/AppConfig/README.md]: @michaelstingl fixed as suggested

**Comment by mneuwert** [comment]
@felix-schwarz may be you could also review and provide feedback to finalise this?

File: MDM/AppConfig/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+## AppConfig how to
+
+### Managed App Config
+Starting with iOS7, Apple has added support for managed application configuration. MDM server can push configuration to the app. The app can the access the configuration using NSUserDefaults class. The configuration is basically a key-value dictionary provided as .plist file.
```
Comment (hosy) [inline MDM/AppConfig/README.md]: Just a typo:
Remove `the` before `access`

The app `can access` the configuration using NSUserDefaults class.

File: MDM/AppConfig/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+### AppConfig XML Schema
+The XML format developed by AppConfig community, makes it easy for developers to define and deploy app configuration. It is not only defining the configuration variables with default value but also provides a configuration UI description which can be interpreted by the tool which generates a plist file. Moreover, specfile XML is consistently supported by major EMM vendors.
```
Comment (hosy) [inline MDM/AppConfig/README.md]: Not sure, but maybe it should be like this:

The XML format developed by AppConfig community, makes it easy for developers to define and deploy `an` app configuration. It is not only defining the configuration variables with default `values` but also provides a configuration UI description which can be interpreted by the tool which generates a plist file.

File: MDM/AppConfig/README.md (hunk 3 above)
Comment (mneuwert) [reply MDM/AppConfig/README.md]: @hosy Thanks for pointing out. Fixed.

File: MDM/AppConfig/README.md (hunk 4 above)
Comment (mneuwert) [reply MDM/AppConfig/README.md]: @hosy Fixed.

**Pull request merged**